src/components/prescriptions/PrescriptionComponent.vue

```diff
@@ -1,7 +1,8 @@
 <template>
     <li class="documentBox">
         <font-awesome-icon :icon="classOfIcon" />
-        <h5 class="documentBox__title">{{ name }} {{ date }}</h5>
+        <h5 class="documentBox__title">{{ name }}</h5>
+        <p class="documentBox__title">{{ date }}</p>
```

Comment:
Is it expected? Youve got two element (other semantic) with same class.  You want to style them with the same selector (they will look both same).

Reply:
ok gut